include/grpc/impl/codegen/port_platform.h

```diff
@@ -189,7 +189,7 @@
 #ifdef __GLIBC__
 #define GPR_POSIX_CRASH_HANDLER 1
```

Comment:
`GPR_POSIX_CRASH_HANDLER` is only used in tests. I don't think it's necessary to have this musl compatibility mode do anything with it. The problem I mentioned with the TCP code also apply to the tests, so most of the tests won't actually be compatible with Alpine even when built with `GPR_MUSL_LIBC_COMPAT` set.